source/atlas-cli-deploy-local.txt

```diff
@@ -275,7 +275,10 @@ Manage a Local Atlas Deployment
 -------------------------------
 
 Use the ``atlas deployments`` command to manage a local |service| 
-deployment. 
+deployment. You can use the following commands for both local and 
+cloud |service| deployments. You can use ``--type local`` or 
+``--type cloud`` to run the command specifically for local or cloud 
```

Comment:
reference: https://preview-mongodbcorryroot.gatsbyjs.io/atlas-cli/DOCSP-34098/command/atlas-deployments-setup/
`Type of deployment that you want to create. Valid values are ATLAS or LOCAL.`
```suggestion
``--type atlas`` to run the command specifically for local or cloud 
```